test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +336,68 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "%s, %s" % [RFauxFactory.gen_alpha, RFauxFactory.gen_alpha]
```

Comment:
this syntax is not commonly used, here are few more ruby-like variants, could you please pick one and change it?
`name = RFauxFactory.gen_alpha + ', ' + RFauxFactory.gen_alpha`
`name = "#{RFauxFactory.gen_alpha}, #{RFauxFactory.gen_alpha}"`